main.py

```diff
@@ -1878,7 +1878,8 @@ async def step(interaction2):
 
         await message.followup.send(embed=embed, view=myview, ephemeral=True)
     else:
-        embed = discord.Embed(title="The Dark Market", description="you already bought from us recenlty. please wait until the next purchase.")
+        hoursago = round((get_cat(message.guild.id, message.user.id, "cataine_active") - int(time.time())) / 3600, 2)
+        embed = discord.Embed(title="The Dark Market", description=f"you bought from us {hoursago} hours ago. please wait until the next purchase.")
```

Comment:
the result is just incorrect ("cataine_active" is a unix timestamp of when cataine will end)

much simpler would be just to
```suggestion
        embed = discord.Embed(title="The Dark Market", description=f"you already bought from us recently. you can do next purchase <t:{get_cat(message.guild.id, message.user.id, 'cataine_active')}:R>.")
```